test.sh

```diff
@@ -111,11 +111,6 @@ fi
 if [[ "$RUN" =~ "fmt" ]] ; then
   start_context "fmt"
   check_gofmt() {
-    # NOTE(@jsha): Go 1.11.5's gofmt and Go 1.12's gofmt don't agree. Let's
```

Comment:
:tada: :worried: :laughing:

Reply:
Excited to add this back in for Go 1.13 :sob: